# The position of the 


std_msgs/Header header

stiffness_commanding/VectorValue[] pairs

